<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accueil</class>
 <widget class="QWidget" name="Accueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(0, 85, 255,0.8);
background-color: rgba(255, 255, 255,0.4);
border-radius: 15%;
</string>
   </property>
   <property name="placeholderText">
    <string>     Searching.......</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-40</y>
     <width>231</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:60px;
background-color: rgb(255, 255, 255);
border:3px black solid;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/build/fichier/icons8_person_128px_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(0, 0, 127,0.2);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>@Admin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:20px;
background-color: rgba(0, 0, 127,0.2);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_home_48px_1.png</normaloff>:/build/fichier/icons8_home_48px_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="history">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(0, 0, 127,0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>History</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_activity_history_64px.png</normaloff>:/build/fichier/icons8_activity_history_64px.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="addr">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(0, 0, 127,0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_add_100px_1.png</normaloff>:/build/fichier/icons8_add_100px_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="editr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(0, 0, 127,0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   Edit or 
Remove Car</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_edit_52px.png</normaloff>:/build/fichier/icons8_edit_52px.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(0, 0, 127,0.2);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_logout_rounded_left_60px.png</normaloff>:/build/fichier/icons8_logout_rounded_left_60px.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="panier">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/build/fichier/icons8_shopping_cart_48px.png</normaloff>:/build/fichier/icons8_shopping_cart_48px.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>scrollArea</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>home</zorder>
  <zorder>history</zorder>
  <zorder>addr</zorder>
  <zorder>editr</zorder>
  <zorder>logout</zorder>
  <zorder>panier</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
